schema/commands.proto

```diff
@@ -1,293 +1,69 @@
-
-  }
-}
-
-message Wallet {
-
-  enum Premissions {
-    add = 0; // permission to add assets to this wallet
-    remove = 1; // permission to remove wallet
-    transfer = 2; // permision to transfer
-  }
-
-  // create wallet with given asset, by default it will have amount = 0.0
-  // Wallet creator "account" must have presmission to create a wallet
-  message Create {
-    string wallet_id = 1;
-    // Full asset name like asset_name#domain
-    string asset_name = 2;
-  }
-
-  // account who created transaction ("creator") must have permission to close this wallet
-  message Close {
-    string wallet_id = 1;
-  }
-
-  // Assign ownership of wallet "wallet_id" to accout "account_id" with permissions
-  message AttachAccount {
-    string account_id = 1;
-    string wallet_id = 2;
-    uint32 permissions = 3; // Logical or of permissions
-  }
-
-  // Remove ownership of wallet "wallet_id" to accout "account_id" with permissions
-  message DetachAccount {
-    string account_id = 1;
-    string wallet_id = 2;
-  }
-
-  // Add amount of asset of the wallet with id "wallet_id"
   message Add {
-    string wallet_id = 1;
-    uint64 amount = 2;
-  }
-
-  // Subtract amount of asset of the wallet with id "wallet_id"
-  message Subtract {
-    string wallet_id = 1;
-    uint64 amount = 2;
-  }
-
-  // Transfer from one wallet to another
-  // Wallet can have only one asset type.
-  // If wallet sender asset is not the same as in reciever - Exchange will be executed
-  message Transfer {
-    string sender_wallet_id = 1;
-    string reciever_wallet_id = 2;
-    uint64 amount = 3;
-  }
-}
-
-
-message Domain {
-
-  // create new domain
-  message Create {
-
-    // full domain name, soramitsu.co.jp for example
-    string domain = 1;
-
-    // if domain is open, then any user can join the domain.
-    // in closed domains accounts can be added only by invite
-    bool isOpen = 2;
-  }
-
-  // note: you can remove only empty domains (without assets and accounts)
-  message Remove {
-    string domain = 1; // full domain name
-  }
-
-  // usage example ( | is logical or):
-  // uint32 p = createAsset | createSubdomain | inviteAccounts
-  enum Permissions {
-    noPermissions = 0;
-    createAsset = 1;
-    createSubdomain = 2; // create
-    inviteAccounts = 4; // invite other accounts to this domain
-    givePermissions = 8; // account with this permission is able to give any permission to other account
-    removeAccountsFromDomain = 16; // account with this permission is able to remove any other account from a domain
-  }
-
-  // transaction "creator" invites account with account_id to the domain
-  message InviteAccount {
-    string account_id = 1; // invitee
-    string domain = 2; // invite to this domain
-    uint32 permissions = 3; // logical or of Permissions enum
-    string username = 4; // account will join with this username: <username>@soramitsu.co.jp
+    string account_name = 1;
+    string asset_name = 2;
+    string amount = 3;
```

Comment:
We should decide how to make it better, it could be included in 'string amount'. In order to speed up development I use string.